### ISSUE TITLE
team 4 , warning message  ,dog cat challenge 

### ISSUE THREAD

**Issue by MohmedAAK** [MohmedAAK/-Advanced-computer-vision-v1#5]
@mohamedgamal1455 
in cell 10 , your monitor is looking for val_acc which is updated in new kears to val_accuarcy 

learning_rate_reduction = ReduceLROnPlateau(monitor='val_acc', 
                                            patience=2, 
                                            verbose=1, 
                                            factor=0.5, 
                                            min_lr=0.00001)
warning appear in fitting cycle  
WARNING:tensorflow:Learning rate reduction is conditioned on metric `val_acc` which is not available. Available metrics are: loss,accuracy,val_loss,val_accuracy,lr
you do not need to re-run or change version ,just keep it in mind ... and approve that you understand , i will close issue then

**Comment by mohamedalswaify** [comment]
@MohmedAAK 
Thank you, I have modified it now. Please look again

**Comment by MohmedAAK** [comment]
@mohamedgamal1455 
Ok I reviewed it 
You didn't not re-run the code but you did understand that you need to change the val_acc to the right word of new version of the library tensorflow 

I will close issue , if there are any misunderstanding reopen it again

**Comment by mohamedalswaify** [comment]
@MohmedAAK 
Yes I did what you said I didn't run the code again because of the internet
